lib/emojiController.js

```diff
@@ -0,0 +1,52 @@
+import { fabric } from 'fabric';
+
+class Emoji {
+  constructor(options = {}) {
+    this.options = options;
+    this.emojiAdded = false;
+    this.position = null;
+    this.options.fontSize = this.options.fontSize || 48;
+    const emoji = String.fromCodePoint(parseInt(this.options.iconCode.substr(2), 16));
+
+    if (!this.emojiAdded) {
```

Comment:
How is this if working?

Reply:
Just making sure that we won't add 2 emojis with one click. Had that kind of bug while testing the controller. On line 31 we set `this.emojiAdded` to `true`. In order to give another emoji we have to create new layer, but this one won't add another emoji to the canvas.